src/components/App/LoadSaveFunctionality.js

```diff
@@ -103,10 +103,10 @@ class LoadSaveButtonBase extends PureComponent {
                         />
                     </DialogContent>
                     <DialogActions>
-                        <Button onClick={() => this.handleClose(true)} color="primary">
+                        <Button onClick={() => this.handleClose(true)} color="white">
                             Cancel
                         </Button>
-                        <Button onClick={() => this.handleClose(false)} color="primary">
+                        <Button onClick={() => this.handleClose(false)} color="white">
```

Comment:
I'd like to keep these as `primary` when in light mode. You can conditionally determine this using the  `isDarkMode` helper function: `import { isDarkMode } from '../../helpers';`